### PR TITLE
Footer section changes

### PR DESCRIPTION
**Pull request by abhijith27**

### CODE REVIEW

**Comment by abhijith27** [comment]
image

File: index.html [hunk 1]
```diff
@@ -557,14 +557,67 @@ <h1 class="pa-h1-v3 pa-bright fadeInDelay03Duration10">Send us a <strong>Message
...
+                                <i class="fas fa-circle fa-stack-2x"></i>
+                                <i class="fab fa-instagram fa-stack-1x"></i>
+                            </a>
+                        </span>
+                        <span class="fa-stack">
+                            <a href="https://www.linkedin.com/company/brainchild-services/" rel="noopener"
+                                target="_blank">
+                                <i class="fas fa-circle fa-stack-2x"></i>
+                                <i class="fab fa-linkedin fa-stack-1x"></i>
+                            </a>
```
Comment (wandercode) [inline index.html]: remove the above icons . not getting seen in website.

**Review by wandercode — APPROVED** [review]

**Comment by abhijith27** [comment]
Removed Icons
image

**Review by rohithkrajan — APPROVED** [review]
LGTM

**Pull request merged**